Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+VERSION ?= 0.0.0
```

Comment:
Note, i believe this should always be set to the latest released version of the operator, when we do a release, we bump this appropriately as part of that process, and commit it to main. Currently we do not have a release of this operator, so it's set to 0.0.0. Version here is the version of the OLM bundle, and must always be a semver, and can't be set to 'latest'.
Once we have an initial version, you can still install the latest version of the operator built from main, but you would need to override this value:
```
make deploy-olm VERSION=0.0.0 
```
We should also setup CI to build and publish this index image on 'main' changes so it is always kept in sync.

@guicassolato I know you made similar, but slightly different, changes for authorino, be good to get your opinion here and see if we can make this consistent across all operator repos.